doc/qgsquick.dox

```diff
@@ -23,6 +26,9 @@ text. See also QgsQuickScaleBarKit.
 \subsubsection qgsquick_overview_widgets_messagelog MessageLog
 A simple panel which can be used for publishing logs messages to a user such as basic information about the application or its status.
 See also QgsQuickMessageLogModel.
+\subsubsection qgsquick_overview_widgets_photocapture PhotoCapture
```

Comment:
not sure if photo capture can be used outside feature panel. I would mention it in FeatureForm docs